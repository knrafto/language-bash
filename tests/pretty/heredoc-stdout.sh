cat <<EOF > /dev/null
Hello world!
EOF
