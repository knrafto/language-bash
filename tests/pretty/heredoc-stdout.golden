cat <<EOF
Hello world!
EOF
>/dev/null;